src/app/common/templates/config-data.yaml

```
@@ -12,6 +12,19 @@ WeakEnums:
     - RadioFaultEnum
     - StepMode
 
+EnumsNotUsedInDataModelStruct:
```

```suggestion
EnumsNotUsedAsTypeInXML:
```
Because the "struct" bit is misleading: events, command payloads, AttributeAccessInterface Write methods, would all do the unknown value thing.  The key part is that this type never goes through DataModel::Decode.

And the good news is that if someone changes that this would fail to compile, right?  That's good.

---

Yeah that is one of the positive side effects is that if someone did change they would get a failure since the `EnsureKnownEnumValue` is removed